src/content/1.7/faq/upgrade.md

```diff
@@ -18,3 +19,15 @@ First, identify what is the `slug` of the Back Office pages. You can find them i
 Second, identify the _Role_ of the User you use to browse the Back Office.
 
 Third, check whether the table `ps_access` grants access to the Back Office pages, using the identifier of the role and the identifiers of the Back Office pages. There must be a record for the role and the page. If there is no such record, create it to grant access.
+
+## Upgrade fails with Error message JqXHR
+
+If during upgrade process, it fails with the error message:
+
+```
+[Ajax/Server operation backup file error] textStatus: "Error" errorThrown: "" JqXHR: "",
```

Comment:
@matks it's not only about backup, right?
```suggestion
[Ajax/Server operation [..]] textStatus: "Error" errorThrown: "" JqXHR: "",
```

Reply:
You're right, I copied too much